docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -230,6 +259,74 @@ Template style
 
       {{foo}}
 
+  Put each ``{% load foo %}`` on it's own line
+
+  Do this:
+
+  .. code-block:: html+django
+
+      {% load foo %}
+      {% load bar %}
+
+  Don't do this:
+
+  .. code-block:: html+django
+
+      {% load foo bar %}
+
```

Comment:
Is this really something that we want to recommend? There is nothing wrong in loading multiple libraries at once. I'd remove it.

Reply:
And we do this in Django itself.

Reply:
Fair enough. I proposed the one-per-line rule to reduce diff noise, but it is a bit overbearing. Perhaps we could instead request the list be sorted alphabetically? That makes it a bit easier to skim.